frontend/src/components/RequestRideModal/RequestRideInfo.tsx

```diff
@@ -270,9 +279,15 @@ const RequestRideInfo = ({
             aria-labelledby="pickupLabel pickupTime"
             ref={register({
               required: true,
-              validate: (pickupTime) => {
+              validate: (pickupTime: string) => {
                 const startDate = getValues('startDate');
-                return startDate ? isTimeValid(startDate, pickupTime) : true;
+                const pickup = errors.pickupTime
+                  ? moment(`${startDate} ${errors.pickupTime.ref.value}`)
```

Comment:
whats the purpose for this conditional statement?

Reply:
I was having trouble with getting the pickup time to be used when there was an invalid choice selected beforehand, but after removing the conditional and using the pickup time itself, it seems to still work properly. I'll change it for both the pickup and drop off times and add the same to the admin ride model and push the changes